packages/flutter/lib/src/cupertino/sheet.dart

```diff
@@ -232,6 +241,10 @@ class CupertinoSheetTransition extends StatefulWidget {
   /// Used to respond to a drag gesture.
   final bool linearTransition;
 
+  /// The gap between the top of the screen and the top of the sheet as a ratio
+  /// of the screen height.
```

Comment:
Should have more documentation here explaining this property, similar to what you wrote for showCupertinoSheet.